vendor/bat-native-ledger/src/bat/ledger/internal/uphold/uphold_util.h

```diff
@@ -51,6 +51,8 @@ void OnWalletStatusChange(LedgerImpl* ledger,
 
 bool ShouldShowNewlyVerifiedWallet();
 
+void VerifyStatusCongruity(const type::ExternalWallet* wallet);
```

Comment:
Nit: maybe another word besides "Congruity"? (Although that is a cool word.)

Some options:

* CheckWalletState
* AssertWalletInvariants
* etc. 

("Verify" is tricky because of the way that we use "verify" with wallets.)

Reply:
`CheckWalletState` works for me. I'll replace 😄

Reply:
Renamed in [d1dd6d6](https://github.com/brave/brave-core/pull/13037/commits/d1dd6d6d33c5e1e243a02794f85b2c31940acb8a)